Desktop_Interface/isobuffer.h

```diff
@@ -18,62 +19,90 @@ class isoDriver;
 class uartStyleDecoder;
 enum class UartParity : uint8_t;
 
-//isoBuffer is a generic class that enables O(1) read times (!!!) on all read/write operations, while maintaining a huge buffer size.
-//Imagine it as a circular buffer, but with access functions specifically designed for isochronous data from an Xmega.
+// isoBuffer is a generic class that enables O(1) read times (!!!) on all
+// read/write operations, while maintaining a huge buffer size.
+// Imagine it as a circular buffer, but with access functions specifically
+// designed for isochronous data from an Xmega.
 
 #define CONSOLE_UPDATE_TIMER_PERIOD (ISO_PACKETS_PER_CTX * 4)
```

Comment:
If possible, changing a lot of these #defines to globals (or variables owned by a specific object and signals sent when the value changes) would be really handy as it would allow the value of these variables to change at runtime.  This would be especially handy for many of the variables in the USB driver as they control frame rates.

Reply:
I am not super familiar with Qt's signal-slot model, but I'll look into it.

Reply:
I can totally change it to a global for now though.